src/jvm/main/org/jetbrains/lincheck/Lincheck.kt

```diff
@@ -87,21 +95,14 @@ object Lincheck {
                             invocations = invocations,
                             verifier = verifier
                         )
-                        throw LincheckAssertionError(failure)
                     }
                     throw LincheckAssertionError(failure)
                 }
             }
         }
     }
 
-    internal const val DEFAULT_INVOCATIONS_COUNT = 50_000
-}
-
-internal data class LincheckSettings(val analyzeStdLib: Boolean) {
-    companion object {
-        val default =  LincheckSettings(analyzeStdLib = true)
-    }
+    internal const val DEFAULT_INVOCATIONS = 50_000
```

Comment:
Should it be consistent with the default number of invocation when testing data structures?